pkg/logs/logs_test.go

```diff
@@ -1,4 +1,5 @@
-//+build !race
+//go:build !race
+// +build !race
```

Comment:
IMO a small PR that does `go fmt ./...` using Go 1.17 would be nice to make this a little easier to review.

Reply:
running this results in no changes, looks like we're already formatted?

Reply:
Ah, I guess this already covered all files that had a build directive, never mind :)